client/i18n/locales/english/translations.json

```diff
@@ -297,6 +297,7 @@
     "help-more": "Help us do more",
     "error": "Something went wrong with your donation.",
     "free-tech": "Your donations will support free technology education for people all over the world.",
+    "no-halo": "If you don't see a gold halo around your profile picture, contact donors@freecodecamp.org.",
```

Comment:
Since stripe is being removed, we don't need to redirect anymore since we are updating the db through webhooks. There are cases where the user is not signed in, and their paypal email does not match their freecodecamp records. In that case, they will not see the halo and would need to email us to update their records. This seems necessary based on the other changes in this pr.

Reply:
I see, thanks for clarifying.